pkg/views/stat/list/view.go

```diff
@@ -0,0 +1,62 @@
+package list
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/models"
+	"github.com/goharbor/harbor-cli/pkg/views/base/tablelist"
+)
+
+var column = []table.Column{
+	{Title: "Public Project", Width: 14},
+	{Title: "Public Repo", Width: 12},
+	{Title: "Private Project", Width: 14},
+	{Title: "Private Repo", Width: 12},
+}
+
+var columnWide = []table.Column{
```

Comment:
donot maintain it separately just keep it with in `column` itself, based on the the `verbose` or `v` flag you can change the length of col